salt/exceptions.py

```diff
@@ -566,3 +566,27 @@ class MissingSmb(SaltException):
     '''
     Raised when no smb library is found.
     '''
+
+
+class NxosError(SaltException):
+    '''
+    NX-OS Base Exception class
+    '''
+
+
+class NxosCliError(NxosError):
+    '''
+    NX-OS Cli Error raised when Cli command rejected by the NX-OS device
+    '''
+
+
+class NxosClientError(NxosError):
+    '''
+    NX-OS Client Error raised for problems connecting to the NX-OS device
+    '''
+
+
+class NxosRequestNotSupported(NxosError):
+    '''
+    Raised for unssuported client requests
```

Comment:
`unsupported`

Reply:
Will fix.

Reply:
I just went ahead and pushed a quick fix.